composer.json

```diff
@@ -34,7 +34,7 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "~3.7",
-        "squizlabs/php_codesniffer": "~1.5"
+        "phpunit/phpunit": "^3.7 || ^4.0",
```

Comment:
PHPUnit 4.0 doesn't work under PHP 5.3, which this component still supports (due to usage in ZF1).